tests/workload/nvidia-smoke/Makefile

```diff
@@ -0,0 +1,25 @@
+.DEFAULT_GOAL:=publish
+
+#### Variables
+REGISTRY ?=
+TAG ?= latest
+REPOSITORY ?= gpu-tests
+
+#### Sanity check we have everything
+ifndef REGISTRY
+$(error "The REGISTRY value must be provided (e.g. REGISTRY=861807767978.dkr.ecr.us-east-2.amazonaws.com make)")
+endif
```

Comment:
Is there a reason a user may not need to specify the `REGISTRY` i.e `kind load`

Reply:
Probably not in the case of this particular workload test, but I see what you're saying. Since this is setting the pattern for future workload tests, I'll update this to check if the variable is set. If so, then it can format the `$REGISTRY/` prefix, and if not it can be empty.